scripts/database-cleaning.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash
set -euo pipefail
```

If you wanted to echo every command before running it, you could do:
```suggestion
#!/bin/bash
set -euox pipefail
```
(that might echo the password to stderr though, so be careful)
```